<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lyout_window</class>
 <widget class="QDialog" name="lyout_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="3">
    <widget class="QPushButton" name="checkAll">
     <property name="text">
      <string>查询全部</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>                   城市查询</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="cityCheck">
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="infor_box">
     <item>
      <property name="text">
       <string>信息</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>姐妹群</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="inforChange">
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLineEdit" name="cityIn"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
